deeppavlov/models/ner/ner.py

```diff
@@ -50,14 +44,11 @@ def __init__(self, **kwargs):
         # Find all parameters for network train
         train_parameters_names = list(inspect.signature(NerNetwork.train_on_batch).parameters)
         # Fill all provided parameters from opt
-        train_parameters = {par: opt[par] for par in train_parameters_names if par in opt}
+        train_parameters = {par: self.opt[par] for par in train_parameters_names if par in self.opt}
         self.train_parameters = train_parameters
 
-        self.opt = opt
-
         # Try to load the model (if there are some model files the model will be loaded from them)
-        super().__init__(save_path=save_path, load_path=load_path,
-                         train_now=train_now, mode=mode)
+        super().__init__(**self.opt)
```

Comment:
Why do you send self.opt to super() and not kwargs? Also, in slotfill.py it's different